### DISCUSSION TITLE
New OpenBHB Dataset on TorchIO

### DISCUSSION THREAD

**Discussion by Duplums** [TorchIO-project/torchio#788] [Q&A]
Hello, 

We have released a new brain MRI dataset (namely OpenBHB) for a challenge that we currently submitted to a NeuroImage special issue. This dataset is open and currently accessible through IEEE dataport (https://ieee-dataport.org/open-access/openbhb-multi-site-brain-mri-dataset-age-prediction-and-debiasing).

As torchIO offers an amazing opportunity to popularize this dataset, we would like to add an interface to this dataset in your library. Currently, the training set contains 3985 T1 images with 3 available pre-processings (views as modalities): Raw, VBM and SBM. The test sets are private for the challenge but it could evolve in the future.

Do you think this dataset is suitable for torchIO ? 

Thank you,
Benoit Dufumier

**Comment by romainVala** [comment]
yes for sure !
looks very interesting, (to me at least)

**Reply by Duplums** [reply]
Nice, how does it work ? Do I have to do a specific pull request or is there another pipeline I am not aware of ?

**Reply by romainVala** [reply]
yes a pull request
@fepegar is the one who may merge it,  (he seems to be less reactive recently ...)
but any way, thanks

**Reply by fepegar** [reply]
Hi! I'm on holiday, I will start replying next week.

**Reply by fepegar** [reply]
I guess I can quickly reply to this one. Please take a look at similar datasets in Torchio, e.g., [RSNAMICCAI](https://torchio.readthedocs.io/datasets.html#rsnamiccai), and feel free to open a PR with an implementation.

**Reply by Duplums** [reply]
Thanks for the quick reply! I'll do the pull request (I hope) next week, following the implementation you recommanded (brats challenge on torchIO).

**Reply by fepegar** [reply]
Great. Feel free to ask any further questions about how to implement it. It should be relatively straightforward.
